<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetalEqualizerFrame</class>
 <widget class="QWidget" name="MetalEqualizerFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#WahWahWidget {color: transparent;}</string>
  </property>
  <widget class="QToasterStompEditLCD" name="lcdDisplay">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>75</y>
     <width>420</width>
     <height>115</height>
    </rect>
   </property>
   <property name="color">
    <enum>QToasterLCD::Yellow</enum>
   </property>
   <property name="currentPage">
    <enum>QToasterLCD::Page1</enum>
   </property>
   <property name="maxPage">
    <enum>QToasterLCD::Page2</enum>
   </property>
   <property name="stompName">
    <string>Wah Wah</string>
   </property>
   <property name="value1Title">
    <string>Low</string>
   </property>
   <property name="value2Title">
    <string>Middle</string>
   </property>
   <property name="value3Title">
    <string>Mid Freq</string>
   </property>
   <property name="value4Title">
    <string>High</string>
   </property>
   <property name="value5Title">
    <string>Low Cut</string>
   </property>
   <property name="value6Title">
    <string>High Cut</string>
   </property>
   <property name="value7Title">
    <string/>
   </property>
   <property name="value8Title">
    <string>Page</string>
   </property>
   <property name="value9Title">
    <string>Mix</string>
   </property>
   <property name="value10Title">
    <string>Ducking</string>
   </property>
   <property name="value11Title">
    <string/>
   </property>
   <property name="value12Title">
    <string>Volume</string>
   </property>
   <property name="value13Title">
    <string/>
   </property>
   <property name="value14Title">
    <string/>
   </property>
   <property name="value15Title">
    <string/>
   </property>
   <property name="value16Title">
    <string>Page</string>
   </property>
   <property name="value17Title">
    <string/>
   </property>
   <property name="value18Title">
    <string/>
   </property>
   <property name="value19Title">
    <string/>
   </property>
   <property name="value20Title">
    <string/>
   </property>
   <property name="value21Title">
    <string/>
   </property>
   <property name="value22Title">
    <string/>
   </property>
   <property name="value23Title">
    <string/>
   </property>
   <property name="value24Title">
    <string/>
   </property>
   <property name="eqBand1Value">
    <double>0.000000000000000</double>
   </property>
   <property name="scaleView">
    <enum>QToasterStompEditLCD::Empty</enum>
   </property>
  </widget>
  <widget class="QStackedWidget" name="smallDials">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>390</width>
     <height>75</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page12">
    <widget class="QToasterDial" name="highCutDial">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>0</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="knobSize">
      <enum>QToasterDial::Small</enum>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::None</enum>
     </property>
     <property name="maxValue">
      <double>16383.000000000000000</double>
     </property>
     <property name="unit">
      <string/>
     </property>
    </widget>
    <widget class="QToasterDial" name="lowCutDial">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="knobSize">
      <enum>QToasterDial::Small</enum>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::None</enum>
     </property>
     <property name="maxValue">
      <double>16383.000000000000000</double>
     </property>
     <property name="unit">
      <string/>
     </property>
    </widget>
    <widget class="QToasterDial" name="dummy7Dial">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>0</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="knobSize">
      <enum>QToasterDial::Small</enum>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::None</enum>
     </property>
     <property name="minValue">
      <double>-5.000000000000000</double>
     </property>
     <property name="maxValue">
      <double>5.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
     <property name="isActive">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page22">
    <widget class="QToasterDial" name="dummy22Dial">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>0</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="knobSize">
      <enum>QToasterDial::Small</enum>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::None</enum>
     </property>
     <property name="maxValue">
      <double>10.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
     <property name="isActive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QToasterDial" name="dummy23Dial">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>0</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="knobSize">
      <enum>QToasterDial::Small</enum>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::None</enum>
     </property>
     <property name="maxValue">
      <double>10.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
     <property name="isActive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QToasterDial" name="dummy21Dial">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="knobSize">
      <enum>QToasterDial::Small</enum>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::None</enum>
     </property>
     <property name="maxValue">
      <double>10.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
     <property name="isActive">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="bigDials">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>560</width>
     <height>100</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page11">
    <widget class="QToasterDial" name="highDial">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::Bi</enum>
     </property>
     <property name="minValue">
      <double>-12.000000000000000</double>
     </property>
     <property name="maxValue">
      <double>12.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
     <property name="unit">
      <string>dB</string>
     </property>
    </widget>
    <widget class="QToasterDial" name="middleDial">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::Bi</enum>
     </property>
     <property name="minValue">
      <double>-12.000000000000000</double>
     </property>
     <property name="maxValue">
      <double>12.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
     <property name="unit">
      <string>dB</string>
     </property>
    </widget>
    <widget class="QToasterDial" name="lowDial">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::Bi</enum>
     </property>
     <property name="minValue">
      <double>-12.000000000000000</double>
     </property>
     <property name="maxValue">
      <double>12.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
     <property name="unit">
      <string>dB</string>
     </property>
    </widget>
    <widget class="QToasterDial" name="midFreqDial">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::Uni</enum>
     </property>
     <property name="minValue">
      <double>0.000000000000000</double>
     </property>
     <property name="maxValue">
      <double>16383.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>1.000000000000000</double>
     </property>
     <property name="precision">
      <UInt>0</UInt>
     </property>
     <property name="unit">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page21">
    <widget class="QToasterDial" name="duckingDial">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::Bi</enum>
     </property>
     <property name="minValue">
      <double>-5.000000000000000</double>
     </property>
     <property name="maxValue">
      <double>5.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
     <property name="isActive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToasterDial" name="mixDial">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::Uni</enum>
     </property>
     <property name="minValue">
      <double>0.000000000000000</double>
     </property>
     <property name="maxValue">
      <double>100.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>1.000000000000000</double>
     </property>
     <property name="unit">
      <string>%</string>
     </property>
    </widget>
    <widget class="QToasterDial" name="dummy19Dial">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::Uni</enum>
     </property>
     <property name="minValue">
      <double>0.000000000000000</double>
     </property>
     <property name="maxValue">
      <double>10.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
     <property name="isActive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QToasterDial" name="volumeDial">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::Bi</enum>
     </property>
     <property name="minValue">
      <double>-5.000000000000000</double>
     </property>
     <property name="maxValue">
      <double>5.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
     <property name="unit">
      <string/>
     </property>
     <property name="isActive">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToasterEnumDial" name="pageDial">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>0</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterEnumDial::Small</enum>
   </property>
   <property name="values">
    <stringlist>
     <string>One</string>
     <string>Two</string>
    </stringlist>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QToasterDial</class>
   <extends>QWidget</extends>
   <header>qtoasterdial.h</header>
  </customwidget>
  <customwidget>
   <class>QToasterEnumDial</class>
   <extends>QWidget</extends>
   <header>qtoasterenumdial.h</header>
  </customwidget>
  <customwidget>
   <class>QToasterStompEditLCD</class>
   <extends>QWidget</extends>
   <header>QToasterStompEditLCD.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pageDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue8(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>443</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pageDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue16(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lowCutDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue5(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>highCutDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue6(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pageDial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>smallDials</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>410</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>highDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue4(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>473</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>middleDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue2(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lowDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue1(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>midFreqDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue3(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mixDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue9(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>duckingDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue10(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volumeDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue12(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pageDial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>bigDials</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>461</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>542</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
